### PR TITLE
Netex spec v2 lines - Updating readings for lines, network, operator

### PR DESCRIPTION
**Pull request by prhod**
Updating readings for lines, network, operator according to PTA validation

### CODE REVIEW

File: src/documentation/netex_idfm_to_ntfs.md [hunk 1]
```diff
@@ -88,53 +95,60 @@ stop_lon | Quay/Centroid/Location | see (1) below
...
 ### networks.txt
-`networks` are provided in the nodes **CompositeFrame/frames/ServiceFrame/Network**. There could be multiple `ServiceFrame` to read.
+`network` objects are provided in the nodes **CompositeFrame/frames/ServiceFrame[]/Network**, each `ServiceFrame` node containing only one `Network` node.
 
 NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
-network_id | *Network/@id* | This field is prefixed. 
-network_name | *Network/Name* | 
+network_id | Network/@id | This field is prefixed. 
+network_name | Network/Name* |
```
Comment (woshilapin) [inline src/documentation/netex_idfm_to_ntfs.md]:
```suggestion
network_name | Network/Name | 
```

File: src/documentation/netex_idfm_to_ntfs.md [hunk 2]
```diff
@@ -154,18 +168,29 @@ other | Bus | Bus | Bus | Bus
...
+`lines` are provided in the nodes **CompositeFrame/frames/ServiceFrame[]/lines/Line[]**. The only `ServiceFrame` containing the lines has an id property set to `STIF:CODIFLIGNE:ServiceFrame:lineid`.
 
 NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
-line_id | *Line/@id* | This field is prefixed. 
-network_id | *Network/@id* | id of the network containing a reference to this line in `Network/members/LineRef/@ref`. This field is prefixed. 
-line_code | *Line/ShortName* | 
-line_name | *Line/Name* | 
+line_id | Line/@id | This field is prefixed. 
+network_id | Network/@id | id of the network containing a reference to this line in `Network/members/LineRef/@ref`. This field is prefixed.
```
Comment (ArnaudOggy) [inline src/documentation/netex_idfm_to_ntfs.md]: These are the old specifications. Network is now defined by Line/RepresentedByGroupRef/@ref, line color and text color are absent, ...

File: src/documentation/netex_idfm_to_ntfs.md [hunk 3]
```diff
@@ -1,17 +1,28 @@
...
 object id is prefixed with a unique prefix (specified for each datasource), following
-the general pattern `<prefix>:<id>`.
+the general pattern `<prefix>:<id>`. This prefix is uniquely defined for all data_sources 
+to make possible safe data aggregations.
+
+### Versions of this document:
+Version | Date | Modification
+--- | --- | ---
+1.0 | 2019-09-13 | Initial redaction
+1.1 | 2019-09-30 | Adding fare zones, MIP access on lines and stop_points, the use of source coordinates of a stop_area (if available), complementary properties of a line (commercial_mode, line_color, line_text_color, note sur ligne)
```
Comment (ArnaudOggy) [inline src/documentation/netex_idfm_to_ntfs.md]: "note on line"

File: src/documentation/netex_idfm_to_ntfs.md (hunk 2 above)
Comment (prhod) [reply src/documentation/netex_idfm_to_ntfs.md]: thanks, it's fixed

**Pull request merged**